src/components/List.svelte

```diff
@@ -39,6 +39,15 @@
   };
 
   function deleteList() {
+    fetch(`/api/card/attachment/delete/list/${list._id}`, {
+      method: "DELETE",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        user_id: $session.user_id,
+      }),
+    });
     $session.lists = $session.lists.filter((l) => {
```

Comment:
Please add a new line here to visually separate the `fetch` from the rest of the code.

```suggestion
    });
    
    $session.lists = $session.lists.filter((l) => {
```